easybuild/easyblocks/g/gromacs.py

```diff
@@ -427,6 +441,13 @@ def install_step(self):
 
                 self.log.info("A full regression test suite is available from the GROMACS web site")
 
+        # Reset installopts for the benefit of the gmxapi extension
+        self.cfg['installopts'] = self.save_installopts
+
+        # Set runtest to None so that the gmxapi extension doesn't try to
+        # run "check" as a command
+        self.cfg['runtest'] = None
```

Comment:
Does this apply now even when gmxapi is not build?

Reply:
Not sure what you really mean here, but resetting it to None doesn't hur.
And this code will be moved around a bit in the new gromacs easyblock I've almost finsihed.